packages/ove-service-imagetiles/src/OVE.Service.ImageTiles/appsettings.json

```diff
@@ -12,10 +12,6 @@
     "MaxConcurrent": 2,
     "PollSeconds": 30
   },
-  "ASyncUploader": {

```

Comment:
Is this configuration change meant to be included in the documentation PR?

Reply:
its an old config variable which is no longer used. might as well get rid of it here rather than make a new PR.